syncstorage/views/util.py

```diff
@@ -89,12 +90,19 @@ def get_resource_timestamp(request):
 
 
 DEFAULT_LIMITS = {}
-DEFAULT_LIMITS["max_request_bytes"] = 1024 * 1024
+DEFAULT_LIMITS["max_record_payload_bytes"] = MAX_PAYLOAD_SIZE
 DEFAULT_LIMITS["max_post_records"] = 100
-DEFAULT_LIMITS["max_post_bytes"] = DEFAULT_LIMITS["max_request_bytes"]
+DEFAULT_LIMITS["max_post_bytes"] = 2 * 1024 * 1024
 DEFAULT_LIMITS["max_total_records"] = 100 * DEFAULT_LIMITS["max_post_records"]
 DEFAULT_LIMITS["max_total_bytes"] = 100 * DEFAULT_LIMITS["max_post_bytes"]
 
+# In production, the request-size limit is actually controlled by nginx.
```

Comment:
I got confused in IRC about this and still am :) Is nginx's size only talking about payloads too?  (Note I've no reason to believe this is wrong, I'm just confused :)

Reply:
I'm pretty sure nginx's size is concerned with total request size, and not payloads.

Reply:
Correct, nginx will basically reject any request with content-length over its configured limit.